JSDemos/Demos/Resizable/Overview/description.md

```diff
@@ -1 +1,21 @@
-The Resizable is a UI component that enables its content to be resizable in the UI.
\ No newline at end of file
+The Resizable enables you to resize its content. In this demo you can drag [handles](/Documentation/ApiReference/UI_Components/dxResizable/Configuration/#handles) to resize the DataGrid.
```

Comment:
(Articles) I can see that the previous version also referenced this component/widget as "The Resizable", but that doesn't match our usual practices. Either you say something like this "The Resizable widget" or you simply say "Resizable" without articles. 

(TextFlow) Your version starts with the Resizable widget and then refers to "its content". That's not immediately clear. (Can I specify it's own content - such as text and images maybe?) And then the second sentence doesn't flow well from the first. I feel that the we better explain this widget if we talk about other elements right away. The developer's goal is to enable resize operation for some other element. If they want it, they use Resizable. So the train of thought goes like this: "you want to let users resize an element" -> "wrap into resizable" -> "this demos shows a data grid example (i.e. locate the grid in the demo)" -> "you can drag handles to try and resize". 

```suggestion
If you want to enable live resize operations for a UI element, wrap that elements into a Resizable widget. In this demo, you can resize the DataGrid. Try to drag the [handles](/Documentation/ApiReference/UI_Components/dxResizable/Configuration/#handles) on the grid's edges.
```